pom.xml

```diff
@@ -47,6 +47,13 @@
       <version>2.0</version>
       <scope>provided</scope>
     </dependency>
+
+    <dependency>
+      <groupId>org.jenkins-ci.modules</groupId>
+      <artifactId>instance-identity</artifactId>
+      <version>2.1</version>
+      <scope>provided</scope>
```

Comment:
Ahh we should do that in BO for user analytics (not to be confused with the other analytics)